pages/vi/vi-planet-installation-and-configuration.md

```diff
@@ -0,0 +1,157 @@
+
+##### Set Up the Repository
+
+- Update the apt package index: `sudo apt-get update`
+- Install packages to allow apt to use a repository over HTTPS
+  ```
+  sudo apt-get install \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    software-properties-common
+  ```
+- Add Docker’s official GPG key: `curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -`
+- Use the following command to set up the stable repository
+  ```
+  sudo add-apt-repository \
+    "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+    $(lsb_release -cs) \
+    stable"
+  ```
+
+##### Install Docker CE
+
+- Update the apt package index: `sudo apt-get update`
+- Install the latest version of Docker CE: `sudo apt-get install docker-ce`
+- Verify that Docker CE is installed correctly by running the hello-world image: `sudo docker run hello-world`
+
+#### Install Docker Compose
+
+Follow the official [Install Docker Compose](https://docs.docker.com/compose/install/#install-compose) guide or the brief rundown below:
+
+- Note the latest release version number at https://github.com/docker/compose/releases/latest. e.g. `1.21.2`. You will need to replace `<latest-release-version-number>` with this in next command.
+- Run this command to download the latest version of Docker Compose:
+  ```
+  curl -L https://github.com/docker/compose/releases/download/<latest-release-version-number>/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
+  ```
+- Apply executable permissions to the binary: `sudo chmod +x /usr/local/bin/docker-compose`
+- Test the installation: `docker-compose --version`
+
+---
+
+## Planet Installation
+
+1. Launch Docker on your machine
+
+1. Go to your OLE project folder
+
+1. Clone the `planet` repository: `git clone https://github.com/open-learning-exchange/docker.git`
+
+1. Go to 'planet/docker' folder: `cd planet/docker`
+
+1. Run the following command to spawn your environment for the **first time**: `docker-compose -f planet.yml -p planet up -d --build`
+
+1. See if the docker containers are running: `docker ps`. You'll see your running container similar to this
+
+  ```
+  CONTAINER ID        IMAGE               COMMAND                  CREATED              STATUS              PORTS                                                                NAMES
+  ea3b914c3193        planetdev_planet    "/bin/sh -c 'bash ..."   About a minute ago   Up 58 seconds       0.0.0.0:3000->3000/tcp                                               planetdev_planet_1
+  57f30698ccda        klaemo/couchdb      "tini -- /docker-e..."   About a minute ago   Up About a minute   4369/tcp, 9100/tcp, 0.0.0.0:2200->5984/tcp, 0.0.0.0:2201->5986/tcp   planetdev_couchdb_1
+  ```
+
+1. See log in action with `docker-compose -f planet.yml -p planet logs -f`, press 'CTRL+C' to exit logs view
+
+---
+
+## Planet Configuration
+
+Go to your community `planet` at <http://localhost:3000>. You will be asked to:
```

Comment:
docker file inside planet has docker running on port `80`

Reply:
For now it will be ok to use `localhost` but when we introduce data syncing with VI then we need use IP Address as replicator does not work with localhost. @paulbert @dogi Shall we ask them to look on IP address? As we have introduced multiple_ips true they should be able to run application on any ip.